doc/README

```diff
@@ -12,24 +18,48 @@ optionally, you might need for special features:
  - gphoto2 (for camera support, recommended)
  - librsvg2 (for watermark plugin)
  - flickcurl (for Flickr support)
- - libGL/sdl12 (to build slideshow viewer)
- - dbus-glib, gnome-keyring (kwallet/gnome keyring support)
- - osm-gps-map for geo tagging view
+ - SDL, SDL-image and Mesa OpenGL (to build darktable-viewer)
+ - osm-gps-map and libsoup for geo tagging view
  - lensfun (lens distortion plugin)
  - OpenEXR for HDR export
-
-other used packages (supplied in the source tree):
-
- - LibRaw nikon_curve (taken from ufraw)
- - RawSpeed
- - squish
+ - flickcurl for uploading direct to Flickr
+ - libsecret for storing passwords
+ - libcolord-dev libcolord-gtk-dev for colour profile support
+ - webp and openjpeg libraries for WebP and JPEG 2000 support
+ - GraphicsMagick library for TIFF EXIF and LDR image format support
```

Comment:
Err, TIFF EXIF is meaningless/wrong.
It should really be something like `TIFF-encoded EXIF thumbnails`